ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/middleware/cimi_params_impl.clj

```diff
@@ -16,7 +16,7 @@
   "Default value for $last parameter.
   When $last is not provided, :last is adjusted
   so that :last - :first + 1 is equals to that default value"
-  20)
+  10000)
```

Comment:
To fix https://github.com/slipstream/SlipStreamServer/issues/683
The code calling the API to gather the mails to send ended using the default value (20).
Details: https://github.com/slipstream/SlipStreamServer/issues/683#issuecomment-205855471